python/kserve/requirements.txt

```diff
@@ -9,14 +9,14 @@ minio>=4.0.9,<7.0.0
 google-cloud-storage>=1.41.1
 adal>=1.2.2
 table_logger>=0.3.5
-numpy~=1.19.2
+numpy~=1.21.5
```

Comment:
this is the value in the latest version of this file in kserve ([link](https://github.com/kserve/kserve/blob/7e74dca4421af26cfbe96dccdf3806e77a63c1b3/python/kserve/requirements.txt#L12))